From 3d0d6731462aba5303977f9696dbb7bc271330f2 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 12 Jun 2019 12:20:32 +0000
Subject: [PATCH 0886/1000] MANTIS 0054609 change: CUSS 1.5 codes.idl:
 DS_TYPES_SSML11, DS_TYPES_KEY_UP, DS_TYPES_KEY_DOWN

codes.idl
V1.4.5
- fixed DS_TYPES_KEY_UP, DS_TYPES_KEY_DOWN


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1463 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 13 ++++++++++---
 1 file changed, 10 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index e522ea9e..e8947cab 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -6,7 +6,7 @@
 //
 //  Date:      31.03.2015
 //
-//  Version:   1.4.4
+//  Version:   1.4.5
 //
 //  Author:    IATA Passenger Experience Management Group: CUWG CUSS-TSG
 //
@@ -88,6 +88,12 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 @li Mantis 0054637: Integration of new device H&S862 SR
 - added data status  DS_TYPES_CYPHER...
 
+@date 12.06.2019
+@li V1.4.5
+@li Mantis 0054609: CUSS 1.5 codes.idl: DS_TYPES_SSML11, DS_TYPES_KEY_UP, DS_TYPES_KEY_DOWN
+- fixed DS_TYPES_KEY_UP, DS_TYPES_KEY_DOWN
+
+
 */
 
 #ifndef CODES_IDL
@@ -214,8 +220,9 @@ module datastatus
     const long DS_TYPES_SSML11                   = 25010; /**< SSML Specification supported by the Announcement Interface */
 
     const long DS_TYPES_KEY                      = 26000; /**< key pressed */
-    const long DS_TYPES_KEY_DOWN                 = 26010; /**< key initially pressed */
-    const long DS_TYPES_KEY_UP                   = 26020; /**< key released */
+    const long DS_TYPES_KEY_UP                   = 26010; /**< Navigation keypad event supported by Keypad Interface */
+    const long DS_TYPES_KEY_DOWN                 = 26020; /**< Navigation keypad event supported by Keypad Interface */
+    
 
     const long DS_TYPES_CYPHER_APP_CERTIFICATE   = 40010; /**< certificate of the application */
     const long DS_TYPES_CYPHER_HW_CERTIFICATE    = 40020; /**< certificate of the hardware */
-- 
2.41.0.windows.1

